### PR TITLE
undo rename

### PR DESCRIPTION
**Pull request by mariaschuld**
**Context:**

Reverts PR #614, since it is giving trouble with plugin tests...

### CODE REVIEW

File: default_config.toml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+[main]
```
Comment (antalszava) [inline default_config.toml:1]: This file was updated in #610 to ensure that no configuration options are being set silently. It was also renamed to `example_config.toml`.

Comment (mariaschuld) [reply default_config.toml]: Sorry @antalszava , I made this PR in a big hurry and should not have done so! I hope it is all fine now!

Comment (antalszava) [reply default_config.toml]: No worries!

**Comment by josh146** [comment]
Tentatively closing this for now, as none of the plugins depend on this enums any more

**Pull request closed without merging**